solr/solr-ref-guide/modules/deployment-guide/pages/solrcloud-shards-indexing.adoc

```diff
@@ -161,6 +161,8 @@ You can delete the old shard at a later time when you're ready.
 
 More details on how to use shard splitting is in the section on the Collection API's xref:shard-management.adoc#splitshard[SPLITSHARD command].
 
+When splitting a shard, a free disk space check is performed on the local file system of the leader shard. This can be disabled through the `solr.shardSplit.checkDiskSpace.enabled` system property (i.e. `-Dsolr.shardSplit.checkDiskSpace.enabled=false`). This property is disabled by default for xref:solr-on-hdfs.adoc[HDFS].
```

Comment:
I don't think this location is a good spot.  It's an overview of a variety of things; not details of how to do something.  Just above there is a link to `shard-management.adoc` which is an appropriate place for this detail.